src/test/java/com/palantir/docker/compose/connection/waiting/SuccessOrFailureMatchers.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.docker.compose.connection.waiting;
+
+import org.hamcrest.Description;
+import org.hamcrest.FeatureMatcher;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import static org.hamcrest.Matchers.anything;
+import static org.hamcrest.Matchers.equalTo;
+
+public enum SuccessOrFailureMatchers {;
```

Comment:
It is usually given as the advice of "use an enum to make static util classes, so that no one can instantiate it!" (obviously missing that you probably should't have static utils classes). We do actually have one here, so why not? An enum is just a class under the scenes, now you literally cannot ever make a `SuccessOrFailureMatchers`, it's like a better private constructor.